react/AppIcon/test/__snapshots__/AppIcon.spec.js.snap

```diff
@@ -27,10 +27,13 @@ exports[`AppIcon component renders default when fetch error occurs 1`] = `
 `;
 
 exports[`AppIcon component renders immediately when icon is alredy preloaded 1`] = `
-<img
-  className="styles__c-app-icon___2_O40"
-  onError={[Function]}
-  src="http://cozy.tools/apps/test/icon"
+<Icon
+  className="styles__c-app-icon___2_O40 styles__c-app-icon-default___3CEmt"
+  color="var(--coolGrey)"
+  height="100%"
+  icon={[Function]}
+  spin={false}
+  width="100%"
 />
 `;
```

Comment:
this test is not accurate anymore since : 
1/ We don't not have any "preloaded" icon 
2/ I think the code doesn't work as expected since `<Icon icon-default` is set when there is an error somewhere or when the "done" status was not set https://github.com/cozy/cozy-ui/blob/4af1a1bcbf8b04050520297beaaba89d7805fc37/react/AppIcon/index.jsx#L121-L133 

Also it seems that we're not displaying anymore a Loading status or else?

Reply:
1/ Le test a été supprimé
2/ Je crois qu'on est bon ici, on affiche bien la `fallbackIcon` en cas d'erreur si elle est présente, sinon la `CubeIcon`. C'est bien ce qu'on veut, non ?
3/ On a toujours notre "loading status" avec un `<div role="progressbar"` quand le statut est `FETCHING`, ça n'a pas changé